.github/workflows/build-bottles.yml

```diff
@@ -1,52 +1,35 @@
 name: Build bottles
 
-on: pull_request
+on:
+  pull_request:
+    paths:
+      - 'Formula/*'
 
 env:
   HOMEBREW_NO_ANALYTICS: 1
   HOMEBREW_NO_AUTO_UPDATE: 1
   HOMEBREW_NO_INSTALL_CLEANUP: 1
 
 jobs:
-  build-linux-bottles:
-    runs-on: ubuntu-latest
-    container:
-      image: homebrew/ubuntu16.04:master
```

Comment:
I'd prefer to use the Docker image to build bottles for Linux. There's [all sorts of stuff](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1604-README.md) installed in the GitHub Actions VM that could be picked up as opportunistic dependencies.

Reply:
If the `container: image: homebrew/ubuntu16.04` line is left in, is it used only for the Linux build?